developer_tests/library/mkmf.template.gfortran

```diff
@@ -0,0 +1,147 @@
+# Template for GNU gfortran on Linux or Mac OSX
+#
+# DART software - Copyright UCAR. This open source software is provided
+# by UCAR, "as is", without charge, subject to all terms of use at
+# http://www.image.ucar.edu/DAReS/DART/DART_download
+#
+# DART $Id$
```

Comment:
```suggestion
```

Reply:
Also need to remove the corresponding lines at the bottom of the file

Reply:
good point, I'll take out all this and the gfortran waffle in the comments. 
I just blindly copied the mkmf.template.gfortran